src/components/ContactCard/ContactCard.jsx

```diff
@@ -0,0 +1,31 @@
+import timeImg from "../../components/img/Icons/time.png";
+
+import "./ContactCard.scss"
+
+export const ContactCard = ({img, alt, title, listLocation, listContacts, time}) => {
+  return (
+    <div className="contacts-card-box">
+      <img className="contacts-card-img" src={img} alt={alt}/>
+      <div className="contacts-card-title">{title}</div>
+      <div className="contacts-card-list-box">
+        <ul className="contacts-card-list-location">
+        {listLocation.map((item) => (
+              <li>{item}</li>
```

Comment:
react requires the 'key' attribute for the map

Reply:
done